core/src/Alert.html

```diff
@@ -73,7 +72,8 @@
 
         return Object.entries(links).reduce((acc, [key, content]) => {
           const elemId = key;
-          const processedData = `<a id="${elemId}">${content.text}</a>`;
+          const escapedText = sanitizeHtml(content.text);
+          const processedData = `<a id="${elemId}">${escapedText}</a>`;
```

Comment:
I would escape/sanitize the key/elemId as well - it won't unlikely be misused (because it will be escaped in the text before so it is not found when we want to replace it with <a> tag - but it won't work then.